src/components/renderer.cpp

```diff
@@ -310,6 +300,24 @@ void renderer::end() {
     fill_background();
   }
 
+
+  if (m_pseudo_transparency) {
```

Comment:
Please document the purpose of this section

Reply:
should be fixed with latest commit